enterprise/server/byte_stream_server_proxy/byte_stream_server_proxy_test.go

```diff
@@ -772,3 +773,269 @@ func BenchmarkWriteUnique(b *testing.B) {
 		}
 	}
 }
+
+func TestWriteCompressibleData_UncompressedToCompressed(t *testing.T) {
+	testDataPath := testfs.RunfilePath(t, "_main/website/yarn.lock")
```

Comment:
ohh this is just some test data? can you call it testdata or something, or just generate it in the test?